src/code/bank20.asm

```diff
@@ -4202,19 +4202,40 @@ func_020_6446::
 
     ret                                           ; $645B: $C9
 
-Data_020_645C::
-    db   $40, $05, $40, $25, $5C, $03, $5E, $03   ; $645C
-
-Data_020_6464::
-    db   $40, $06, $40, $26, $60, $03, $62, $03   ; $6464
-
-Data_020_646C::
-    db   $40, $07, $40, $27, $64, $03, $7E, $03   ; $646C
-
-Data_020_6474::
-    dw   Data_020_645C
-    dw   Data_020_6464
-    dw   Data_020_646C
+GreenTunicStatusOAMData::
```

Comment:
Not sure if this section is overexplained

Reply:
The documentation seems fine to me.

On the formatting side, display lists put the two bytes on the same line (e.g `db $40, OAM_GB_PAL_5 ; comments`, so it could be interesting to use the same format here too.